monitoring/update-bucket-capacity-info-cronjob/alerts.yaml

```diff
@@ -0,0 +1,44 @@
+# Variables which should be replaced. Similar to grafana dashboards' __inputs section
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: update_bucket_capacity_info_cronjob
+    type: constant
+    value: artesca-data-ops-update-bucket-capacity-info
+  - name: update_bucket_capacity_info_job_duration_threshold
+    type: config
+    value: 240
+  - name: update_bucket_capacity_info_success_job_existence_duration_threshold
+    type: config
+    value: 600
+
+groups:
+  - name: update-bucket-capacity-info-cronjob/alerts.rules
+    rules:
+      - alert: UpdateBucketCapacityJobTakingTooLong
+        expr: |
+          sum by(job_name) (kube_job_status_completion_time{job_name=~"${update_bucket_capacity_info_cronjob}.*"})
+          - sum by(job_name) (kube_job_status_start_time{job_name=~"${update_bucket_capacity_info_cronjob}.*"})
+          > ${update_bucket_capacity_info_job_duration_threshold}
```

Comment:
This works, but it waits for the job to finish before triggering an alert. I'm not sure if this is the desired behaviour ?

One idea is to use the following :
```
time() - 
(sum by(job_name) (kube_job_status_failed{job_name=~"${update_bucket_capacity_info_cronjob}.*"})
> sum by(job_name) (kube_job_status_completion_time{job_name=~"${update_bucket_capacity_info_cronjob}.*"})
or sum by(job_name) (kube_job_status_completion_time{job_name=~"${update_bucket_capacity_info_cronjob}.*"}))
> ${update_bucket_capacity_info_job_duration_threshold}
```

This works because ZKOP forbids having multiple concurrent runs of the same job [here](https://github.com/scality/zenko-operator/blob/c392b853d0dd16379062faa316c94dce4387cdfc/pkg/controller/zenko/cronjob_generator.go#L39), this means that if it is time for a new job run and the previous job run hasn't finished yet, the CronJob skips the new job run. Which means that all you have to do is check the time difference between the last job run (that either failed or succeeded) and the current time.

To get the maximum value of two metrics you use the expression :
```

a > b or b
```
Where `a` and `b` are metrics
if `a>b` is true `a` will be returned else `b` is returned